src/main/java/teammates/ui/controller/PageData.java

```diff
@@ -912,27 +908,51 @@ public String getInstructorFeedbackSessionActions(FeedbackSessionAttributes sess
             "data-courseid=\"" + session.courseId + "\" data-fsname=\"" + session.feedbackSessionName + "\" " +
             "data-toggle=\"modal\" data-target=\"#remindModal\">Remind particular students</a></li></ul></div> "
         );
-        
+        String unpublishLinkStyle = "class=\"btn btn-default btn-xs btn-tm-actions session-unpublish-for-test\"";
+        String publishLinkStyle = "class=\"btn btn-default btn-xs btn-tm-actions session-publish-for-test";
+        result.append(getInstructorFeedbackSessionPublishAndUnpublishAction(session, isHome, instructor, publishLinkStyle, unpublishLinkStyle));
+
+        return result.toString();
+    }
+    
+    /**
+     * Returns the link of publish and unpublish action for the session feedback
+     * @param session
+     *         The session details
+     * @param isHome
+     *         Flag whether the link is to be put at homepage (to determine the redirect link in delete / publish)
+     * @param instructor
+     *         The instructor attributes of the session feedback
+     * @param publishLinkStyle
+     *         A string containing the CSS styles of the publish link
+     * @param unpublishLinkStyle
+     *         A string containing the CSS styles of the unpublish link
+     * @return
+     */
+    public String getInstructorFeedbackSessionPublishAndUnpublishAction(FeedbackSessionAttributes session, boolean isHome, InstructorAttributes instructor, String publishLinkStyle, String unpublishLinkStyle) {
```

Comment:
But they have different class attributes in different places. I think trying to define those variables in the function itself will not work. At least we need to pass in one argument to say which type of button we want to create so the correct class attributes can be appended. Or we can define two methods to handle this, but it seems not very good neither. Can you give me some suggestion?